.travis.yml

```diff
@@ -3,10 +3,12 @@ python:
     - "3.4"
 cache: "pip"
 install:
-    - "pip install ."
+    - "pip install .[tests]"
 script:
     - "flake8 ."
-    - "py.test pythoncz_tests"
+    - "pytest --cov-report="
```

Comment:
I can remove the `--cov-report=`, hopefully that wouldn't affect coveralls. I have it there in [fiobank](https://github.com/honzajavorek/fiobank) for some reason, but I don't remember why.